### PR TITLE
Install clojure cli tools to a location common to powershell & pwsh

### PR DESCRIPTION
**Pull request by ikappaki**
Hi,

could you please consider patch to install the Clojure cli tools to a common PowerShell modules location that is discoverable by both the `powershell` and `pwsh` shells. It addresses #61.

This is done by setting the `PSModulePath` variable to the common location before installing the Clojure cli tools on MS-Windows. The Clojure cli tools install script looks into that variable to present the user with  the list of paths to install to, and this action chooses option 1, i.e. this path.

I've generated the js script code by running `npm run all`, and I've also updated the examples in smoke tests and the readme file, not sure if there are more references.

 The smoke tests should pass once committed to master, not sure how to run them from the branch.

Thanks

### CODE REVIEW

**Comment by DeLaGuardo** [comment]
Looks like it is still failing [smoke test](https://github.com/ikappaki/setup-clojure/runs/7446309629?check_suite_focus=true)
Could it be because exec is running a new powershell [here](https://github.com/DeLaGuardo/setup-clojure/pull/62/files#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197R479)? I'm not a windows user so it is hard for me to reason why it is happening

**Comment by ikappaki** [comment]
> Looks like it is still failing [smoke test](https://github.com/ikappaki/setup-clojure/runs/7446309629?check_suite_focus=true)
> Could it be because exec is running a new powershell [here](https://github.com/DeLaGuardo/setup-clojure/pull/62/files#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197R479)? I'm not a windows user so it is hard for me to reason why it is happening

It is failing because the smoke test in my PR branch references the master branch which doesn't have the fix. Would you like to change the smoke testa action to reference the PR branch so that it passes, and before we are about to merge I switch the reference back to master?

Thanks

File: dist/index.js [hunk 1]
```diff
@@ -477,6 +477,8 @@ function setupWindows(version, cmdExeWorkaround, githubAuth) {
         }
         const url = `download.clojure.org/install/win-install${version === 'latest' ? '' : `-${version}`}.ps1`;
         yield exec.exec(`powershell -c "iwr -useb ${url} | iex"`, [], {
```
Comment (DeLaGuardo) [inline dist/index.js:479]: I think using `powershell -c` is redundant here.

```suggestion
        yield exec.exec(`iwr -useb ${url} | iex`, [], {
```

**Comment by DeLaGuardo** [comment]
Amazing! I see it works now.

**Comment by ikappaki** [comment]
> 
> right, sorry, I should check that first :) yes, please check with your repo/branch first

Done, could you please have another look to confirm?

**Review by DeLaGuardo — APPROVED** [review]
Thanks for the PR! It also allows proper caching for ClojureCLI on windows runners :)

**Pull request merged**